packages/web/src/components/pages/Webhooks/WebhookDetails/ActionTab/Headers.tsx

```diff
@@ -48,6 +52,14 @@ const Headers: FC = () => {
       initialValues={{headers: decomposeHeaders(current!.headers ?? {})}}
       disabled={!mayEdit}
       onConfirm={onFinish}
+      footer={
+        <>
+          Learn more about{' '}
+          <ExternalLink href={externalLinks.notificationsAndWebhooks} target="_blank">
```

Comment:
Could you point to HTTP Headers section?

https://docs.testkube.io/articles/webhooks/#uri-and-http-headers